src/modules/fancyzones/lib/LLMouseHookWrapper.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <vector>
+#include <functional>
+
+struct LLMouseEvent
+{
+    bool
+        state;
+    WPARAM wParam;
+    std::function<void()> callback;
+};
+
+class LLMouseHookWrapper
+{
+public:
+    LLMouseHookWrapper(WPARAM, std::function<void()>);
+    void enable();
+    void disable();
+
+private:
+    unsigned int hookNumber;
+    static HHOOK hookProc;
```

Comment:
is it possible to not have static fields by passing `this` to `llMouseHookCallback` in `WPARAM`?